### ISSUE TITLE
Possibility to add Volume for Vault CSI integration

### ISSUE THREAD

**Issue by Aimp91** [stefanprodan/podinfo#308]
Please add the possibility to add Volume to the pod in Order to use Vault CSI to inject secrets.
Thank you

**Comment by stefanprodan** [comment]
You want this in the Helm chart for the `data` volume or is this for something else?

**Comment by Aimp91** [comment]
Yes, on the Helm chart will be great to have the possibility to add deployment.spec.template.spec.volumes and deployment.spec.template.spec.containes.volumeMount.  In this Way secret can be injected using Vault CSI.
Thank you

**Comment by stefanprodan** [comment]
Which secret are you referring to? The redis url?

**Comment by Aimp91** [comment]
Not a specific secret, in order to inject secrets ( any type)  in the pod using Vault CSI, the Helm chart should be configurare to add Volumens on the pod  and VolumeMount on the container.
Thank you in advance
